app/src/main/java/com/github/houseorganizer/houseorganizer/task/FirestoreTask.java

```diff
@@ -35,7 +34,11 @@ public DocumentReference getTaskDocRef() {
     public void changeTitle(String newTitle) {
         super.changeTitle(newTitle);
 
-        taskDocRef.update("title", newTitle);
+        try {
```

Comment:
I think we have been using throw statements instead of try catch for this exceptions

Reply:
I used this to maintain the signature of the method I override - with maybe 30 more mins of work I can make this a non-overriding method.